backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/pporder/PPOrderLinesViewFactory.java

```diff
@@ -155,7 +155,8 @@ private ViewLayout createViewLayout(final WindowId windowId)
 	{
 		return ImmutableList.of(
 				createProcessDescriptorForIssueReceiptWindow(de.metas.ui.web.pporder.process.WEBUI_PP_Order_Receipt.class),
-				createProcessDescriptorForIssueReceiptWindow(de.metas.ui.web.handlingunits.process.WEBUI_M_HU_Pick.class),
```

Comment:
what is `WEBUI_M_HU_Pick` about? is it now obsolete? if yes, please remove it altogether

Reply:
ok, so I guess `WEBUI_M_HU_Pick` will stay in its own right, jsut won't be accessible from the production modal, right?